docs/guides/almalinux/_include/alma-config.rst

```diff
@@ -0,0 +1,25 @@
+Configure Alma
+--------------------------
+
+Set hostname
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block::
+
+  hostname AlmaRocks
+  hostname > /etc/hostname
+
+Fix SELinux filesystem labels
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block::
+
+  fixfiles -F -f relabel
+
+Create initial admin user
```

Comment:
User creation is typically left as an exercise to the end user. Are root logins entirely disabled, such that even setting a password for that account won't let you login?